user_profile/urls.py

```diff
@@ -1,9 +1,7 @@
-from rest_framework.routers import DefaultRouter
+from django.urls import path
 
 from user_profile import views
 
-router = DefaultRouter()
-
-router.register('', views.UserProfileViewSet, basename='UserProfile')
-
-urlpatterns = router.urls
```

Comment:
If want to use router to automatically configure urls, doesn't we have to use module path and include?
like,
`(python)
urlpatterns = [
path("foo/bar",include(router.urls),
]
`?

Reply:
nobody uses like that on official documents or stackoverflow...

well, but I don't know what effect that actually cuz i didn't use like that.